test/shell/test_coverage.sh

```diff
@@ -0,0 +1,27 @@
+# shellcheck source=./test_runner.sh
+dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+. "${dir}"/test_runner.sh
+. "${dir}"/test_helper.sh
+runner=$(get_test_runner "${1:-local}")
+
+test_coverage_on() {
+    bazel coverage \
+          --extra_toolchains="//scala:code_coverage_toolchain" \
+          //test/coverage/...
+    diff test/coverage/expected-coverage.dat $(bazel info bazel-testlogs)/test/coverage/test-all/coverage.dat
+}
+
+test_coverage_includes_test_targets() {
+    bazel coverage \
+          --extra_toolchains="//scala:code_coverage_toolchain" \
+          --instrument_test_targets=True \
+          //test/coverage/...
+    grep -q "SF:test/coverage/TestAll.scala" $(bazel info bazel-testlogs)/test/coverage/test-all/coverage.dat
+}
+
+xmllint_test() {
```

Comment:
This seems not used in this file. Did you mean to copy it over from `test/shell/test_misc.sh`?

Reply:
Thanks for catching!